src/main/java/net/dv8tion/jda/internal/requests/restaction/pagination/ThreadChannelPaginationActionImpl.java

```diff
@@ -59,19 +61,16 @@ protected String getPaginationLastEvaluatedKey(long lastId, ThreadChannel last)
         if (useID)
             return Long.toUnsignedString(lastId);
 
-        if (order == PaginationOrder.FORWARD && lastId == 0)
-        {
-            // first second of 2015 aka discords epoch, hard coding something older makes no sense to me
-            return "2015-01-01T00:00:00.000";
-        }
-
-        // this should be redundant, due to calling this with PaginationAction#getLast() as last param,
-        // but let's have this here.
-        if (last == null)
+        if (last != null)
+            // If threads were retrieved
+            // OffsetDateTime#toString() is defined to be ISO8601, needs no helper method.
+            return last.getTimeArchiveInfoLastModified().toString();
+        else if (lastId != 0)
+            // If the user skipped ahead
+            return TimeUtil.getTimeCreated(lastId).toString();
```

Comment:
I'm not sure if this should even be allowed, it's very easy for the user to try to skip using an ID, when in reality it uses the ID *as a iso8601 timestamp*, requiring to pass the thread archival timestamp, instead of the thread ID.

My use case was to retrieve a channel by ID by making a small window with the pagination, would be *much* better if JDA had a way to retrieve a channel by ID